### ISSUE TITLE
[BUG] - Color options in CircularProgress not working

### ISSUE THREAD

**Issue by c0d3x** [nextui-org/nextui#2764]
### NextUI Version

2.3.2

### Describe the bug

When using all color example in my code fro: [](https://nextui.org/docs/components/circular-progress#colors)

There is an issue with not showing correctly:
![image](https://github.com/nextui-org/nextui/assets/661911/f1dd5fdb-eaba-4ed8-b701-d40835cec937)

Colors should work fine, since I use them also for Avatar which works fine:
![image](https://github.com/nextui-org/nextui/assets/661911/b2370f45-2a3a-47a9-90b7-a6d2f5579a8d)

### Steps to Reproduce the Bug or Issue

Use latest NextJS + NextUI components, and copy example code from NextUI.

### Expected behavior

Expected component to work proparly with colors.

### Operating System Version

macOS latest

### Browser

Chrome

**Comment by wingkwong** [comment]
@c0d3x Can you provide a Stackblitz link? It looks fine on my side.

![image](https://github.com/nextui-org/nextui/assets/35857179/b9758328-60c4-4154-93fe-5a270c911c48)

**Comment by c0d3x** [comment]
Here I seem to have same issue:
https://stackblitz.com/edit/stackblitz-starters-2gqjvm

**Comment by wingkwong** [comment]
Thanks for the link. Seems something wrong when a custom color is defined. I will take a look.

**Comment by c0d3x** [comment]
@wingkwong Thanks for looking into this!
I tried adding the extend, however the color are same for light and dark mode. That seems a bit odd.

How can I get the colors to change when in dark mode? 
Seems like they all stay same on light/dark mode

**Comment by c0d3x** [comment]
By adding these objects it seems to work for all except default.
Not sure why the default key is not taking the changes.

```TS
        nextui({
            themes: {
                dark: {
                    colors: {
                        danger: {
                            DEFAULT: '#E31F04',
                            foreground: '#FFFFFF',
                        },
                        default: {
                            DEFAULT: '#0B0B26',
                            foreground: '#FFFFFF',
                        },
                        primary: {
                            DEFAULT: '#A5B1AA',
                            foreground: '#FFFFFF',
                        },
                        secondary: {
                            DEFAULT: '#FEFBE6',
                            foreground: '#0B0B26',
                        },
                        success: {
                            DEFAULT: '#55D440',
                            foreground: '#FFFFFF',
                        },
                        warning: {
                            DEFAULT: '#FCB27C',
                            foreground: '#FFFFFF',
                        },
                    },
                },
 ```

**Comment by wingkwong** [comment]
> I tried adding the extend, however the colors of CircularProgress are the same for light and dark mode.

does other components work with light and dark mode? wondering if the setup is correct here or not.

> By adding these objects it seems to work for all except default.
> Not sure why the default key is not taking the changes.

can you show some examples?

**Comment by c0d3x** [comment]
> > I tried adding the extend, however the colors of CircularProgress are the same for light and dark mode.
> 
> does other components work with light and dark mode? wondering if the setup is correct here or not.
> 
> > By adding these objects it seems to work for all except default.
> > Not sure why the default key is not taking the changes.
> 
> can you show some examples? 
> 

Yes the Avatar component works with default theme changes, but not CircularProgress

**Comment by spavelsit** [comment]
`<CircularProgress className='circular-progress' color='warning' />`
The following solution helped
`
.circular-progress {
	.text-warning {
		--tw-text-opacity: 1;
               /* Replaced color with stroke */
		stroke: hsl(
			var(--nextui-warning) /
				var(--nextui-warning-opacity, var(--tw-text-opacity))
		);
	}
}
`

**Comment by wingkwong** [comment]
Closing in favor of v3. If the issue is reproducible in v3, please create a new one with a minimal reproducible example.
